assets/js/types.ts

```diff
@@ -1,5 +1,10 @@
 import { useToast } from "@chakra-ui/react";
 
+export interface BasicUserInfo {
```

Comment:
I don't understand the purpose of this type. I guess that the idea was to narrow Client arguments to only the one that are used but in this case what is the purpose of the "name" argument?

Reply:
It's basically an intersection between Client and Presenter (and possibly other similar structures like Participant) that was requested by Radek in [this comment](https://github.com/membraneframework-labs/membrane_live/pull/97#discussion_r1017504514).
It is used for functions like `isDeviceEnabled` that were specific only for Client but shouldn't be so.
Other alternatives include passing just the email address, but 
1. the `alias` type would only be an alias for a `string`
2. The `name` information can be helpful in future.

Reply:
There is also an option of putting all the types like `Client`, `Participant` etc. into one type that is a set sum of them.  AFAIC this is how the solution from Videoroom new frontend

Reply:
Change the name to User and change types to interfaces, `Participant` and other types should extend User.